data/nomis.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Nomis",
+  "description": "Nomis represents the Reputation Layer for web3, specifically designed to empower users to access and leverage their wallet history. It serves as a valuable tool for BUIDLers, enabling accelerated growth through active engagement and incentives for genuine contributors and early adopters.",
+  "short_description": "Nomis Score represents your Onchain Reputation within the Starknet ecosystem.",
```

Comment:
@unchase 

The short_description needs to be a bit shorter than this to fit into the preview / list views. I suggest

"Your on-chain Starknet reputation"